.github/workflows/ci-experimental.yml

```diff
@@ -106,4 +106,9 @@ jobs:
           key: ${{ runner.os }}-vendor-modules-${{ steps.submodules.outputs.hash }}
 
       - name: Run tests
-        run: make V=1 LOG_LEVEL=DEBUG QUICK_AND_DIRTY_COMPILER=1 test2 testwakunode2
+        run: |
+          if [ ${{ runner.os }} == "Linux" ]; then
```

Comment:
Hey, people use https://github.com/abiosoft/colima as a workaround to the docker and mac image licensing issues, do you think we can try the same as well? 
non-blocker, just curious!

Reply:
Morning @rymnc ! thanks so much for the suggestion. I take a note to try it in the future when I have more time.